datumaro/plugins/imagenet_format.py

```diff
@@ -27,19 +28,42 @@ def __init__(self, path):
 
     def _load_categories(self, path):
         label_cat = LabelCategories()
-        for images_dir in os.listdir(path):
-            label_cat.add(images_dir)
+        for images_dir in sorted(os.listdir(path)):
+            if images_dir != ImagenetPath.IMAGES_DIR_NO_LABEL:
+                label_cat.add(images_dir)
         return { AnnotationType.label: label_cat }
 
     def _load_items(self, path):
         items = {}
+        labels = {}
         for images_dir in os.listdir(path):
-            for image_path in glob(osp.join(path, images_dir, '*.jpg')):
-                image_name = osp.splitext(osp.basename(image_path))[0]
-                label_id = self._categories[AnnotationType.label].find(images_dir)[0]
+            if images_dir == ImagenetPath.IMAGES_DIR_NO_LABEL:
+                for image_path in glob(osp.join(path, images_dir, '*.jpg')):
```

Comment:
I think, we should avoid hardcoding `.jpg`. Instead, we can try to load a file, check https://github.com/openvinotoolkit/datumaro/blob/develop/datumaro/plugins/image_dir.py#L31